packages/product-components/src/components/CoinLogo/coins.ts

```diff
@@ -7,7 +7,7 @@ export const COINS: Record<NetworkSymbol | LegacyNetworkSymbol, string> = {
     ada: require('../../images/coins/ada.svg'),
     base: require('../../images/coins/base.svg'),
     bch: require('../../images/coins/bch.svg'),
-    bnb: require('../../images/coins/bnb.svg'),
+    bsc: require('../../images/coins/bnb.svg'),
```

Comment:
These files should be renamed as well for consistency

Reply:
updated